python/src/opendp/polars.py

```diff
@@ -400,13 +402,112 @@ def wrap(*args, **kwargs):
                     out = attr(*args, **kwargs)
 
                     # re-wrap any lazy outputs to keep the conveniences afforded by this class
-                    if isinstance(out, (_LazyGroupBy, _LazyFrame)):
+                    if isinstance(out, _LazyFrame):
                         return LazyFrameQuery(out, query)
+                    
+                    if isinstance(out, _LazyGroupBy):
+                        return LazyGroupByQuery(out, query)
 
                     return out
 
                 return wrap
             return attr
+        
+
+        def sort(  # type: ignore[empty-body]
+            self,
+            by: IntoExpr | Iterable[IntoExpr],
+            *more_by: IntoExpr,
+            descending: bool | Sequence[bool] = False,
+            nulls_last: bool | Sequence[bool] = False,
+            maintain_order: bool = False,
+            multithreaded: bool = True,
+        ) -> LazyFrameQuery:
+            """Sort the LazyFrame by the given columns."""
```

Comment:
```suggestion
            """Sort the ``LazyFrame`` by the given columns."""
```
and similarly below.